cloud/jenkins/pgo_operator_gke_alpha.groovy

```diff
@@ -0,0 +1,297 @@
+void CreateCluster(String CLUSTER_PREFIX) {
```

Comment:
Minor thing: CLUSTER_PREFIX is actually used as cluster suffix

Reply:
https://github.com/Percona-Lab/jenkins-pipelines/pull/1009/commits/5e753a5a785f2f85ea63895d0d7eccf57bfcd61f